### PR TITLE
Add missing reference to Materialized-View extension.

### PR DESCRIPTION
**Pull request by yurmix**

### CODE REVIEW

File: docs/content/development/extensions.md [hunk 1]
```diff
@@ -95,6 +95,7 @@ All of these community extensions can be downloaded using [pull-deps](../operati
 |kafka-emitter|Kafka metrics emitter|[link](../development/extensions-contrib/kafka-emitter.html)|
 |druid-thrift-extensions|Support thrift ingestion |[link](../development/extensions-contrib/thrift.html)|
 |druid-opentsdb-emitter|OpenTSDB metrics emitter |[link](../development/extensions-contrib/opentsdb-emitter.html)|
+|materialized-view-selection materialized-view-maintenance|Materialized View|[link](../development/extensions-contrib/materialized-view.html)|
```
Comment (jihoonson) [inline docs/content/development/extensions.md]: Would you please add a comma between them? Or perhaps itemizing them is better? Not sure it's possible in the table.

Comment (yurmix) [reply docs/content/development/extensions.md]: Done.

Comment (yurmix) [reply docs/content/development/extensions.md]: BTW, I kept the space as well for beautification reasons (Here, it creates two lines in the same cell).

Comment (yurmix) [reply docs/content/development/extensions.md]: I'll keep it like that, but BTW it is possible to itemize:

|Name|Description|Docs|
|----|-----------|----|
|<ul><li>materialized-view-selection</li><li>materialized-view-maintenance</li></ul>|Materialized View|[link](../development/extensions-contrib/materialized-view.html)|

Comment (jihoonson) [reply docs/content/development/extensions.md]: Cool thanks!

**Pull request merged**